packages/dx-react-grid-demos/src/demo-sources/grid-editing/cell-editing-tab-with-edit-column.jsxt

```diff
@@ -27,18 +30,23 @@ export default () => {
     columnValues: { id: ({ index }) => index, ...defaultColumnValues },
     length: 8,
   }));
+  const [editingCells, setEditingCells] = useState([]);
+  const [selectTextOnEditStart] = useState(true);
```

Comment:
We can just write 
<TableInlineCellEditing selectTextOnEditStart />
and get rid of this variable